tests/support/kubernetes.py

```diff
@@ -239,7 +251,12 @@ async def create_namespaced_custom_object(
             plural,
             body,
         )
-        assert body["api_version"] == f"{group}/{version}"
+        # Asyncio Kubernetes does not do camelCase to snake_case conversion,
+        # but it eventually probably should.  In the meantime, accomodate it.
+        try:
+            assert body["api_version"] == f"{group}/{version}"
+        except KeyError:
+            assert body["apiVersion"] == f"{group}/{version}"
```

Comment:
The mock should exactly match the behavior of the thing it's mocking, so if kubernetes_asyncio doesn't do camel-case conversion, neither should we. In other words, my previous code was just buggy (sorry about that; if it weren't, you would have caught this problem earlier).

Reply:
removed